package-lock.json

```diff
@@ -16060,6 +15265,7 @@
       "version": "17.7.2",
       "resolved": "https://registry.npmjs.org/yargs/-/yargs-17.7.2.tgz",
       "integrity": "sha512-7dSzzRQ++CKnNI/krKnYRV7JKKPUXMEh61soaHKg9mrWEhzFWhFnxPxGl+69cD1Ou63C13NUPCnmIcrvqCuM6w==",
+      "dev": true,
```

Comment:
[These seemed a bit odd on scan. I checked the template proposal and it seems to also be doing a large change and setting dev to true on a bunch of things.](https://github.com/ministryofjustice/hmpps-template-typescript/pull/353/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519R8728)